{"message":"Object of type UUID is not JSON serializable","level":"ERROR","service":"api","timestamp":"2021-07-22 19:29:25,729"}
Traceback (most recent call last):
  File "/home/app/main.py", line 77, in upload_file
    json.dumps(task_data))
  File "/usr/local/lib/python3.9/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/local/lib/python3.9/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.9/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/local/lib/python3.9/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type UUID is not JSON serializable
